plugins/audit_logs/scripts/cleanup.py

```diff
@@ -0,0 +1,26 @@
+"""
+This script is for deleting old audit log entries from the database. Example invocation:
+
+    girder shell --plugins=audit_logs cleanup.py -- --days=30 --types=rest.request
+"""
+
+import click
+import datetime
+from girder.plugins.audit_logs import Record
+
+
+@click.command()
```

Comment:
Untested, but I think if this plugin were to have a `setup.py` it could use the `girder.cli_plugins` entrypoint and be invoked with something like `girder audit-logs cleanup`.

Reply:
In a previous incarnation of this PR, I had a comment at the top of the script that said something like "this can be restructured after pip installable plugins lands", and I think that still holds; it'll be more enticing to move to that once we're forced to have a setup.py anyway.